docs/src/main/sphinx/release/release-381.md

```diff
@@ -0,0 +1,102 @@
+* Add support for fault-tolerant execution with exchange spooling on Azure Blob Storage . ({issue}`12211`)
+* Increase the default number of stages allowed for a query from 100 to 150,
+  specified with `query.max-stage-count`, so large queries do not fail. ({issue}`12292`)
+* Use the `fault_tolerant_execution_partition_count` session property, or the
+  `fault-tolerant-execution-partition-count` configuration property, to adjust
+  the number of partitions for distributed joins and aggregations executed with
+  fault-tolerant execution enabled (`retry-policy=TASK`). ({issue}`12263`)
+* Rename configuration property used for selecting a policy for low memory query
+  killer from `query.low-memory-killer.policy` to
+  `query.low-memory-query-killer.policy`. ({issue}`12242`)
+* Introduce specific configuration property
+  `query.low-memory-task-killer.policy` for selecting a policy for low memory
+  task killer. The policy is used if task level retries are enabled
+  (`retry-policy=TASK`). ({issue}`12242`)
+* Introduce `least-wasted` low memory task killer policy. This policy avoids
+  killing tasks that are already executing for a long time, so a significant
+  amount of work is not wasted.  ({issue}`12242`)
+* Improve Advanced Encryption Standard (AES) processing performance on ARM64 processors. ({issue}`12251`)
+* Fix potential planning failure of queries with multiple subqueries.  ({issue}`12199`)
+
+## Security
+
+* Add support for automatic discovery of OpenID Connect metadata with OAuth 2.0
+  authentication. ({issue}`9788`)
+* Re-introduce `ldap.ssl-trust-certificate` as legacy configuration to avoid
+  update failures. ({issue}`12187`)
+* Fix query failure when a table has multiple column masks defined. ({issue}`12262`)
+* Fix regression in file-based system and connector access controls. ({issue}`12204`)
+* Fix regression in LDAP password authenticator usage with ActiveDirectory. ({issue}`12231`)
+
+## Web UI
+
+* Ensure consistent sort order in the list of workers. ({issue}`12290`)
+
+## CLI
+
+* Add automatic suggestions from command history. This can be disabled with the
+  `--disable-auto-suggestion` option. ({issue}`11671`)
+
+## BigQuery connector
+
+* Allow skipping view materialization via `bigquery.skip-view-materialization`
+  configuration  property. ({issue}`12210`)
+
+## ClickHouse connector
+
+* Prevent data loss when non-transactional insert fails. ({issue}`12229`)
+
+## Druid connector
+
+* Prevent data loss when non-transactional insert fails. ({issue}`12229`)
+
+## Elasticsearch connector
+
+* Improve query performance by simplifying filters sent to Elasticsearch. ({issue}`10717`)
+* Fix failure when reading timestamp values with `row` type that are not ISO
+  8601 formatted. ({issue}`12250`)
+
+## Iceberg connector
+
+* Add support for {doc}`/sql/update`. ({issue}`12026`)
+* Fix potential query failure or incorrect results when reading Iceberg table
+  containing equality deletion files. ({issue}`12026`)
```

Comment:
What's an "equality deletion" file?

Reply:
@alexjo2144 and @findepi might have to help .. I read a bunch of the conversation in the PR but now sure how to translate that to a meaningful enough release notes entry

Reply:
"Equality Delete Files" should be, see https://iceberg.apache.org/spec/#equality-delete-files

Reply:
Cool .. I reworded and link there..